takwimu/takwimu_ui/src/components/Profile/index.js

```diff
@@ -43,6 +43,9 @@ class Profile extends React.Component {
       tabs
     } = profile;
     const { value } = this.state;
+
+    // https://facebook.github.io/create-react-app/docs/using-global-variables
+    const pageDescription = window.takwimu.data_topic_page_description;
```

Comment:
Put this in the `profileData.description` object as done with the other `window` variables. See: https://github.com/TakwimuAfrica/Takwimu/blob/570cecc2385ff9f7b14fd733e93e2996e1a04b83/takwimu/takwimu_ui/src/index.js#L25